src/test/app/forms/models/name.ts

```diff
@@ -11,8 +11,8 @@ describe('Name', () => {
 
   describe('constructor', () => {
     it('should set the primitive fields to undefined', () => {
-      const companyName = new Name()
-      expect(companyName.text).to.be.undefined
+      const organisationName = new Name()
```

Comment:
I don't think this should be `organisationName`. It should be `name` only. I now it's legacy and IDE converted this.